doc/server/contents/conf.rst

```diff
@@ -837,3 +850,102 @@ For example::
 
         return request
 
+==============
+Token revocation
+==============
+        
+In order to enable the token revocation endpoint a dictionary with key `token_revocation` should be placed 
+under the `endpoint` key of the configuration.
+
+If present, the token revocation configuration should contain a `policy` dictionary
```

Comment:
Is the `policy` key needed? or is a policy applied by default when the key is missing?

```suggestion
If present, the token revocation configuration may contain a `policy` dictionary
```